YourScheduler/YourScheduler.Infrastructure.xUnitTests/RepositoriesTests/TeamRoleRepositoryTests.cs

```diff
@@ -1,12 +1,114 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+using YourScheduler.Infrastructure.Entities;
+using YourScheduler.Infrastructure.Repositories;
 
 namespace YourScheduler.Infrastructure.xUnitTests.RepositoriesTests
 {
-    internal class TeamRoleRepositoryTests
+    public class TeamRoleRepositoryTests
     {
+        private readonly Team testTeam = new()
```

Comment:
Dawaj jeszcze jakieś brzegowe zdarzenia. Narazie to tylko Happy path.

Reply:
Nie mogłem ich testować, więc narazie bare minimum